pkg/ui/workspaces/cluster-ui/src/api/databasesApi.ts

```diff
@@ -9,7 +9,7 @@
 // licenses/APL.txt.
 
 import {
-  executeInternalSql,
+  executeInternalSql, LARGE_RESULT_SIZE,
```

Comment:
I think this being on the same line may be a lint error

Reply:
Done.